Dockerfile

```diff
@@ -44,6 +44,8 @@ RUN apt-get update \
  && gem install --no-document bundler \
  && rm -rf /var/lib/apt/lists/*
 
+RUN apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false
```

Comment:
Can you move this change to a different commit since its not related to the 4.0 migration?  Also, add a comment describing what its doing.